app/cells/folio/console/form/header/show.slim

```diff
@@ -24,4 +24,4 @@
       == cell('folio/console/index/tabs', options[:tabs])
 
   - if model.try(:object).try(:errors).present?
-    == cell('folio/console/form/errors', model)
+    == cell('folio/console/form/errors', model, options.slice(:hide_btn_for))
```

Comment:
header componenta by mela mit popisnejsi option nez `hide_btn_for`, kdyz se to tyka chyb